src/test/run-pass/macros/macro-at-most-once-rep-2015.rs

```diff
@@ -0,0 +1,33 @@
+// run-pass
+#![allow(unused_mut)]
+// The logic for parsing Kleene operators in macros has a special case to disambiguate `?`.
+// Specifically, `$(pat)?` is the ZeroOrOne operator whereas `$(pat)?+` or `$(pat)?*` are the
+// ZeroOrMore and OneOrMore operators using `?` as a separator. These tests are intended to
+// exercise that logic in the macro parser.
```

Comment:
This is not true any more. `?` is not allowed as a separator. `$(pat)?+` matches `pat +` or `+`, _not_ `pat`, `pat ? pat`, etc